src/backends/automate.js

```diff
@@ -6,7 +6,14 @@ import requestApiBase from '../utils/request-api';
 import createBrowserstackStatus from '../utils/create-browserstack-status';
 import * as ERROR_MESSAGES from '../templates/error-messages';
 
-const API_POLLING_INTERVAL = 80000;
+// This is done to prevent idle timeout happening
+// in automate sessions, as network can become bottleneck.
+// Try reducing the number in case network is unable to
+// send commands in the given amount of time
+const envAPIPoll = process.env['API_POLL'];
+const API_POLLING_INTERVAL = !isNaN(envAPIPoll) ?
+                            parseInt(envAPIPoll, 10) :
+                            80000;
```

Comment:
I suggest checking the value if it is lesser than [BROWSERSTACK_IDLE_TIMEOUT](https://www.browserstack.com/automate/timeouts) and emit a warning with [process.emitWarning](https://nodejs.org/api/process.html#process_process_emitwarning_warning_type_code_ctor) in this case.

Reply:
Done